### ISSUE TITLE
Request for new release

### ISSUE THREAD

**Issue by UebelAndre** [google/cargo-raze#304]
I think given recent changes a new release should be made. However, I'd really like it if the release could be made after https://github.com/google/cargo-raze/pull/303 is merged. Additionally, there are a few PRs before it and there are some other PRs in work that might be worth including as well. But that's up to the code owners (obviously) 😄

**Comment by UebelAndre** [comment]
The PR linked in this issue has been merged and I feel the changes I made to the repo should be in a decent spot for a release. That said though, I've opened up #319 which I feel really completes the UX for cargo workspace support and it'd be nice to have that be the first user experience

**Comment by UebelAndre** [comment]
@acmcarther Up to you though. I would otherwise have 3 PRs (including #319) I'd like to get through for a release. #319 being the largest of the 3.

**Comment by UebelAndre** [comment]
@acmcarther Actually, I think a release now would be best since it'd unblock some efforts in rules_rust.

**Comment by acmcarther** [comment]
```
./publish-new-version.sh 0.8.0
```

**Comment by UebelAndre** [comment]
Thank you!

**Comment by UebelAndre** [comment]
Gonna close this for now to make it cleaner as to what went into this. I'll open a new request when I think a new release is ready.
